src/modules/homePage/load-homepage.js

```diff
@@ -0,0 +1,63 @@
+import ShowsList from './fetchMovie.js';
+import postLike from './postLike.js';
+import countItem from './CountItem.js';
+import renderReservePopup from '../reserve/load-reserve-pop.js';
+import showModal from '../comment/popup.js';
+
+const show = new ShowsList();
+
+const cardContainer = document.getElementById('card-container');
+const displayCards = (response) => {
+  const section = document.createElement('section');
+  section.classList.add('items');
+  countItem();
+  section.innerHTML += `
+  <div class="row-1">
+    <img src="${response.image.medium}" alt="movie posture">
+  </div>
+  <div class="row-2">
+    <h3 class="title-movie">${response.name}</h3>
+    <p class="flex-col">
+      <button class="heart">
+        <i class="fa fa-heart heart-icon" id="${response.id}"></i>
+      </button><span class=like-count>0</span>likes
+    </p>
+  </div>
```

Comment:
- [ ] Good job team. Please notice here the like counter should not be static, I notice you are correct in sendiong an update to your involvement API when I click the like button. But when the page reloads the changes are lost. Please make sure when the user clicks on the like button the data is recorded inside the involvement API and when we reload the page we gate the same amount of likes we already make.

Reply:
Retain number of likes when page reloads.